src/features/Application/ManageApplicationContainer.tsx

```diff
@@ -130,49 +145,57 @@ class ManageApplicationContainer extends React.Component<
   }
 
   public render() {
-    const { mode, hackerDetails, submitted } = this.state;
+    const { mode, hackerDetails, submitted, pageNumber } = this.state;
     return submitted ? (
       <Redirect to={FrontendRoute.HOME_PAGE} />
     ) : (
-      <MaxWidthBox m={'auto'} maxWidth={'500px'}>
-        <Sidebar
-          currentPage="Application"
-          status={this.state.hackerDetails.status}
-          confirmed={true}
-        />
-        <Helmet>
-          <title>
-            {mode === ManageApplicationModes.CREATE ? 'Create' : 'Edit'}
-            Application | McHacks 6
-          </title>
-        </Helmet>
-        <MaxWidthBox maxWidth={'500px'} m={'auto'}>
-          <H1
-            color={'#F2463A'}
-            fontSize={'30px'}
-            textAlign={'left'}
-            marginTop={'0px'}
-            marginBottom={'20px'}
-            marginLeft={'0px'}
-          >
-            {mode === ManageApplicationModes.CREATE ? 'Create' : 'Edit'} your
-            Application
-          </H1>
-          <FormDescription>{CONSTANTS.REQUIRED_DESCRIPTION}</FormDescription>
+      <HorizontalSpacer paddingLeft={'20%'}>
+        <MaxWidthBox m={'auto'} maxWidth={'500px'}>
+          <Sidebar
+            currentPage="Application"
+            status={this.state.hackerDetails.status}
+            confirmed={true}
+          />
+          <Helmet>
+            <title>
+              {mode === ManageApplicationModes.CREATE ? 'Create' : 'Edit'}
+              Application | McHacks 6
```

Comment:
Can the page title here be abstracted to the constants file? i.e. 

`Application | {CONSTANTS.HACKATHON_NAME}`

Reply:
different task already made